src/utils/mod.rs

```diff
@@ -432,6 +433,21 @@ pub fn parse_quotes(s: impl AsRef<str>) -> Vec<String> {
     args
 }
 
+pub fn parse_webhook(url: &Url) -> Option<(u64, &str)> {
```

Comment:
This should be given documentation. For example:
```suggestion
/// Parses the ID and token from a webhook URL. 
pub fn parse_webhook(url: &Url) -> Option<(u64, &str)> {
```
You should probably also add an example.

Reply:
Done.